doc/classes/VisualScriptDeconstruct.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptDeconstruct" inherits="VisualScriptNode" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		A Visual Script node which deconstructs a base type instance into its parts.
```

Comment:
Grammatically correct, but you use both "which" and "that" in similar sentences from different doc files (eg. VisualScriptEmitSignal). I'd advise to choose one.

Reply:
Let's say I will stay with "which".